lib/marked_section_deleter.rb

```diff
@@ -0,0 +1,60 @@
+require "services"
+
+class MarkedSectionDeleter
+  def initialize(logger = STDOUT)
```

Comment:
Minor: It feels slightly odd/confusing calling this a "logger" when it's actually just an output stream.